README.md

```diff
@@ -3,9 +3,30 @@
 
 # Xpring Common JavaScript
 
-Xpring Common JavaScript Library provides common JavaScript functionality to all client side libraries in [Xpring SDK](https://github.com/xpring-eng/xpring-sdk). This library is built to be consumed as a dependency to other libraries, rather than a standalone library.
+Xpring Common JavaScript Library provides common JavaScript functionality to all client side libraries in [Xpring SDK](https://github.com/xpring-eng/xpring-sdk) ([TS/JS version](https://github.com/xpring-eng/xpring-js)). This library is built to be consumed as a dependency to other libraries, rather than a standalone library.
 
-Developers probably want to use one of the language specific libraries for [Xpring SDK](https://github.com/xpring-eng/xpring-sdk#client-side-libraries), rather than consuming this library directly. However, developers with specific requirements for interacting with Xpring SDK may want to depend on this library.
+Developers probably want to use one of the language specific libraries for [Xpring SDK](https://github.com/xpring-eng/xpring-sdk#client-side-libraries) ([TS/JS version](https://github.com/xpring-eng/xpring-js)), rather than consuming this library directly. However, developers with specific requirements for interacting with Xpring SDK may want to depend on this library.
+
+## Getting started
```

Comment:
There's a bit in [CONTRIBUTING.md](https://github.com/xpring-eng/xpring-common-js/blob/master/CONTRIBUTING.md#building-the-library) about how to build the library. However, you're not the first person to have this problem, so our current model of where to find the build instructions is clearly wrong. 

Would you consider either:
1) Linking that section here
2) Removing the section from CONTRIBUTING.md and putting it here

I'll propagate whatever changes you make to this file structure to the other Xpring SDK libraries as well.